### DISCUSSION TITLE
✅ Sign when widget has custom css

### DISCUSSION THREAD

**Discussion by bombarda-rpd** [elementor/elementor#21597] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Sometimes I add some CSS code in the widget's custom css, in some cases just for testing purposes. This CSS can be very difficult to find later on.

### Describe the solution you'd like

My suggestion would be to put an asterisk in the browser widgets.

**Comment by loucasl** [comment]
You can see elements that have custom CSS applied in the navigator/structure panel. There is a colour indicator which when you hover shows that there is custom CSS applied to that element.

**Comment by bombarda-rpd** [comment]
Thanks for the reply!

In fact you are right. 👍

Thanks
Em 9/03/2023, 15:43 +0000, Loucas ***@***.***>, escreveu:

**Comment by nicholaszein** [comment]
Hi @bombarda-rpd!

Thank you for contributing! 🙏 

This feature is already available in the Structure Panel (Navigator) as @loucasl correctly pointed out. It was added 5 years ago in version 2.6.0.

Please, make sure you visited the [Help Center](https://elemn.to/help) before submitting your requests. Elementor is a feature-rich builder, and chances are that what you're looking for already exists.

Kind regards
